lib/generator/baseGenerator.py

```diff
@@ -291,13 +292,13 @@ def output_suite_result(global_config, index_config, exec_config, output_result_
         except Exception as e:
             logger.error('Cannot move statistics file to result folder. Error: {exp}'.format(exp=e))
 
-        # copy current result file to result folder
+        # move current result file to result folder
```

Comment:
running the same test in next round will affect the test result if we don't move it to the result folder